kafka-agent/src/main/java/io/strimzi/kafka/agent/KafkaAgent.java

```diff
@@ -167,13 +185,18 @@ boolean handleBrokerState() {
                     ready = true;
 
                 } else if (i++ % 60 == 0) {
-                    LOGGER.debug("Metric {} = {} (type: {})", brokerStateName, value, value.getClass());
+                    LOGGER.debug("Metric {} = {} (type: {})", brokerStateName, observedState, observedState.getClass());
```

Comment:
I assume you mean if `brokerState.value()` returns a null value? I can't seem to see whether that can return a null value or not since we are using an abstract type here

Reply:
You interpreted by managed comment correctly.   I'd suggest erring on the side of caution and being null safe.